apps/src/StudioApp.js

```diff
@@ -783,6 +794,10 @@ StudioApp.prototype.resizeHeaders = function (fullWorkspaceWidth) {
   var toolboxWidth = 0;
   var showCodeWidth = 0;
 
+  var clearPuzzleHeader = document.getElementById('clear-puzzle-header');
+  var clearPuzzleWidth = clearPuzzleHeader
+      ? clearPuzzleHeader.getBoundingClientRect().width : 0;
```

Comment:
I thought we added an exception :)

"Binary and Ternary line continuations should have the operator at the beginning of the line: (note that this is different from Google’s JavaScript style guide)"

https://docs.google.com/a/code.org/document/d/1N18MR7rY3gASdSPGOC0sKISNXxyoGGS59i3JWbteWOY/edit#heading=h.yzziwdeuup14

I'm open to either style.